src/Symfony/Component/Validator/Constraints/BicValidator.php

```diff
@@ -63,7 +70,11 @@ public function validate($value, Constraint $constraint)
         }
 
         // next 2 letters must be alphabetic (country code)
-        if (!ctype_alpha(substr($canonicalize, 4, 2))) {
+        if (!class_exists(Intl::class)) {
+            throw new RuntimeException('The "symfony/intl" component is required to use the Bic constraint.');
```

Comment:
should be a `LogicException` as it involves to fix your code by adding the missing dependency (discussion currently happening in https://github.com/symfony/symfony/pull/28513)